justspacesapp/templates/base.html

```diff
@@ -0,0 +1,43 @@
+{% load staticfiles %}
+
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <!-- <link rel="icon" href="{% static 'images/favicon.jpg' %}"> -->
+
+    <!-- JS -->
+    <script
+    src="https://code.jquery.com/jquery-2.2.4.min.js"
```

Comment:
Let's host a copy of the jquery.js, rather than using the CDN (then, we need not worry about ever losing access to the CDN). You can run `curl https://code.jquery.com/jquery-2.2.4.min.js` to get it.

Reply:
Oh, maybe related (?), what's that integrity property do?